zhaquirks/__init__.py

```diff
@@ -136,9 +141,16 @@ def _calculate_battery_percentage(self, raw_value):
         elif volts > self.MAX_VOLTS:
```

Comment:
Should we just do ?
```
volts = max(volts, self.MIN_VOLTS)
volts = min(volts, self.MAX_VOLTS)
```

Reply:
one liner would work too, but less readable

Reply:
Sure

Reply:
Done.